hw_5/HW_pandas_matplotlib.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import math
+import matplotlib.ticker as tck
+
+
+# In[2]:
+
+
+plt.rcParams['font.size'] = '20'
+plt.rcParams['figure.figsize'] = (20, 10)
+
+
+# # Task1
+
+# In[3]:
+
+
+def read_gff(path):
+    df = pd.read_csv(path, delimiter='\t', header=1, names=['chromosome', 'source','type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
+    return df
```

Comment:
1. Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
2. `df` - плохое имя для датафрейма, стоит дать осмысленное название по содержимому
```suggestion
    gff_data = pd.read_csv(path, delimiter='\t', comment="#", names=['chromosome', 'source','type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
    return gff_data
```